Cargo.toml

```diff
@@ -14,3 +14,7 @@ rustc_version = "0.2.3"
 
 [dev-dependencies]
 doc-comment = "0.3"
+
+[features]
+default = []
+memoffset_constant = []
```

Comment:
Maybe call this `const_experimental` or `unstable_const` or so? The `memoffset` part is not needed as features are scoped per-crate, but we should make it clear that this feature is exempt from stability guarantees.